pages/radar.md

```diff
@@ -7,6 +7,9 @@ permalink: /radar.html
 
 [/home](/)
 
+## [GitHub’s Engineering Team has moved to Codespaces](https://github.blog/2021-08-11-githubs-engineering-team-moved-codespaces/)
```

Comment:
Improving software quality is a key goal of the Code Quality Foundation, which is achieved, among other things, by using various tools, such as continuous integration, code checkers, automatic testing utilities, and so on.  Configuring and supporting such tools for large projects can take significant resources (including money). Purchasing and maintaining servers gets quite expensive! The use of cloud technologies, where all the necessary tools are already configured and provided to developers (even directly from an IDE), is an obvious path of least resistance for the project manager. Therefore, we need to focus on developing tools that could work with cloud technologies, such as plugins, to find and fix defects in the source code.